### DISCUSSION TITLE
Sliders can't be set accurately

### DISCUSSION THREAD

**Discussion by mondalaci** [home-assistant/frontend#22872] [Configuration]
Consider the following slider. It's a struggle to set an exact value.

![image](https://github.com/user-attachments/assets/4d453820-ae3c-45ec-bae4-d7b775b6f90b)

Even if I open the property's pop-up window, it's still hard to pick exact values. I have to click/tap numerous times in the slider to pick a value close to the one I want to set. 

![image](https://github.com/user-attachments/assets/de4d583b-b623-4c4e-b5e8-e69ab22bcbeb)

At the very least, when I click/tap on the numeric value in the property popup (3.5m below), I should be able to type the desired value.

**Comment by alanv72** [comment]
mode: box

works for most of my entities that have this issue.

**Comment by mondalaci** [comment]
How to achieve what you're suggesting through the GUI?
